templates/error/general_http_error.json.twig

```diff
@@ -0,0 +1,13 @@
+{
+    "error": {
+        {% block title %}
+        "location": "{{ pageTitle|json_encode }}",
+        {% endblock %}
+        {% block statusCode %}
+        "code": {{ statusCode|json_encode }},
+        {% endblock %}
+        {% block message %}
+        "message": "{{ errorMessage|default("Please try your request again, check the server logs, or contact the site administrator for support")|xl|json_encode}}"
```

Comment:
remove surrounding double quotes from line 4 and line 10

Reply:
Fixed.